DigitalLearningSolutions.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -23,6 +24,22 @@ public static class HtmlHelperExtensions
                 : string.Empty;
         }
 
+        public static string GetSelectedCssClassIfTabSelected(
+            this IHtmlHelper htmlHelper,
+            NavMenuTab currentNavMenuTab,
+            string selectedCssClass = "selected"
+        )
+        {
+            if (!(htmlHelper.ViewContext.ViewData["SelectedTab"] is NavMenuTab selectedTab))

```

Comment:
Could we extract "SelectedTab" and "ApplicationType" into constants on a static class which will hold all the ViewData keys which Layout expects to see? We should probably do the same for the other keys used by Layout for completeness, but put a comment on the ones that people shouldn't be using going forwards.

Reply:
Where exactly should I put comments?